linetools/analysis/absline.py

```diff
@@ -531,3 +567,47 @@ def Wr_from_N_transition(N, transition, llist='ISM'):
     # return
     return Wr_from_N(N, wrest, fosc)
 
+
+def N_from_Wr_transition(Wr, transition, llist='ISM'):
```

Comment:
While over-loading input statements is a bit dangerous, I might
prefer it here, i.e. combine these 2 methods and the 2nd input
can either be a Quantity (wrest) or a str (transition).  Could even
allow an AbsLine.